src/netaddress.h

```diff
@@ -120,354 +120,354 @@ static constexpr uint16_t I2P_SAM31_PORT{0};
  */
 class CNetAddr
 {
-    protected:
```

Comment:
This is the whitespace fixup that was dropped as a standalone change in https://github.com/bitcoin/bitcoin/pull/22562. Here it is combined with some other changes that touch those files.

`netaddress.h` is not fully clang-formatted. Running `clang-format -i netaddress.h` produces the following diff:

<details>
<summary>format diff</summary>

```diff
diff --git i/src/netaddress.h w/src/netaddress.h
index 77e1c0464e..eb35ed3fac 100644
--- i/src/netaddress.h
+++ w/src/netaddress.h
@@ -39,14 +39,13 @@ static constexpr int ADDRV2_FORMAT = 0x20000000;
  * belongs to both `NET_UNROUTABLE` and `NET_IPV4`.
  * Keep these sequential starting from 0 and `NET_MAX` as the last entry.
  * We have loops like `for (int i = 0; i < NET_MAX; ++i)` that expect to iterate
  * over all enum values and also `GetExtNetwork()` "extends" this enum by
  * introducing standalone constants starting from `NET_MAX`.
  */
-enum Network
-{
+enum Network {
     /// Addresses from these networks are not publicly routable on the global Internet.
     NET_UNROUTABLE = 0,
 
     /// IPv4
     NET_IPV4,
 
@@ -70,22 +69,20 @@ enum Network
     NET_MAX,
 };
 
 /// Prefix of an IPv6 address when it contains an embedded IPv4 address.
 /// Used when (un)serializing addresses in ADDRv1 format (pre-BIP155).
 static const std::array<uint8_t, 12> IPV4_IN_IPV6_PREFIX{
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xFF, 0xFF
-};
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xFF, 0xFF};
 
 /// Prefix of an IPv6 address when it contains an embedded TORv2 address.
 /// Used when (un)serializing addresses in ADDRv1 format (pre-BIP155).
 /// Such dummy IPv6 addresses are guaranteed to not be publicly routable as they
 /// fall under RFC4193's fc00::/7 subnet allocated to unique-local addresses.
 static const std::array<uint8_t, 6> TORV2_IN_IPV6_PREFIX{
-    0xFD, 0x87, 0xD8, 0x7E, 0xEB, 0x43
-};
+    0xFD, 0x87, 0xD8, 0x7E, 0xEB, 0x43};
 
 /// Prefix of an IPv6 address when it contains an embedded "internal" address.
 /// Used when (un)serializing addresses in ADDRv1 format (pre-BIP155).
 /// The prefix comes from 0xFD + SHA256("bitcoin")[0:5].
 /// Such dummy IPv6 addresses are guaranteed to not be publicly routable as they
 /// fall under RFC4193's fc00::/7 subnet allocated to unique-local addresses.
```
</details>

I guess a scripted diff that fully formats the files would be even easier to verify and more complete.

Reply:
Thanks, taken. Couldn't get a scripted-diff working, but happy to take if you find one.

Reply:
```
commit f8a3771957cb1d4d202cd7e3173edfbbeed2657e
Parent: faaf9e930d94328c6be42516beab817d45d847f1
Author:     Vasil Dimov <vd@FreeBSD.org>
AuthorDate: Thu Jul 29 12:39:49 2021 +0200
Commit:     Vasil Dimov <vd@FreeBSD.org>
CommitDate: Thu Jul 29 12:39:49 2021 +0200


    scripted-diff: violate guidelines
    
    -BEGIN VERIFY SCRIPT-
    clang-format -i src/netaddress.h src/test/fuzz/deserialize.cpp
    -END VERIFY SCRIPT-

diff --git a/src/netaddress.h b/src/netaddress.h
index c81f9fd5d8..eb35ed3fac 100644
--- a/src/netaddress.h
+++ b/src/netaddress.h
...
same diff as in fa81dae3db
...
```

Verify:
```sh
$ test/lint/commit-script-check.sh origin/master..HEAD
Running script for: f8a3771957cb1d4d202cd7e3173edfbbeed2657e
clang-format -i src/netaddress.h src/test/fuzz/deserialize.cpp
OK
```